src/pages/workspace/expensifyCard/WorkspaceExpensifyCardPageEmptyState.tsx

```diff
@@ -0,0 +1,68 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import {View} from 'react-native';
+import FeatureList from '@components/FeatureList';
+import type {FeatureListItem} from '@components/FeatureList';
+import * as Illustrations from '@components/Icon/Illustrations';
+import useLocalize from '@hooks/useLocalize';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type {FullScreenNavigatorParamList} from '@libs/Navigation/types';
+import WorkspacePageWithSections from '@pages/workspace/WorkspacePageWithSections';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+
+const expensifyCardFeatures: FeatureListItem[] = [
+    {
+        icon: Illustrations.MoneyReceipts,
+        translationKey: 'workspace.moreFeatures.expensifyCard.feed.features.cashBack',
+    },
+    {
+        icon: Illustrations.CreditCardsNew,
+        translationKey: 'workspace.moreFeatures.expensifyCard.feed.features.unlimited',
+    },
+    {
+        icon: Illustrations.MoneyWings,
+        translationKey: 'workspace.moreFeatures.expensifyCard.feed.features.spend',
+    },
+];
+type WorkspaceExpensifyCardPageEmptyStateProps = {route: StackScreenProps<FullScreenNavigatorParamList, typeof SCREENS.WORKSPACE.EXPENSIFY_CARD>['route']};
+
+function WorkspaceExpensifyCardPageEmptyState({route}: WorkspaceExpensifyCardPageEmptyStateProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {shouldUseNarrowLayout} = useResponsiveLayout();
+
+    return (
+        <WorkspacePageWithSections
+            shouldUseScrollView
+            icon={Illustrations.HandCard}
+            headerText={translate('workspace.common.expensifyCard')}
+            route={route}
+            guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_EXPENSIFY_CARD}
+            shouldShowOfflineIndicatorInWideScreen
+        >
+            <View style={[styles.mt3, shouldUseNarrowLayout ? styles.workspaceSectionMobile : styles.workspaceSection]}>
+                <FeatureList
+                    menuItems={expensifyCardFeatures}
```

Comment:
For these screen items, the text gets truncated on a smaller screen which caused issue https://github.com/Expensify/App/issues/45901